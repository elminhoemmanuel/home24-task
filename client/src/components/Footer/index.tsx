import React from 'react'
import classes from './Footer.module.css';

const Footer = () => {
    return (
        <div className={classes.footer}>
            Alle Preise sind in Euro (â‚¬) inkl. gesetzlicher Umsatzsteuer und Versandkosten.
        </div>
    )
}

export default Footer