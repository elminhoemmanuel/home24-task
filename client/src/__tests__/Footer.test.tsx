import React from 'react';
import { render, screen } from '@testing-library/react';
import { within } from '@testing-library/dom'
import Footer from '../components/Footer';


test("On render, check that Footer is rendered correctly", () => {
    const { getByTestId } = render(<Footer />);
    const { getByText } = within(screen.getByTestId('footer'))
    expect(getByText("Alle Preise sind in Euro (â‚¬) inkl. gesetzlicher Umsatzsteuer und Versandkosten.")).toBeInTheDocument()
});
