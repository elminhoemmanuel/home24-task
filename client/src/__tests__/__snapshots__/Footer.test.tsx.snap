// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`On render, check that Footer is rendered correctly 1`] = `
<div>
  <button
    class="text-sm rounded text-white p-3 cursor-pointer bg-hmred hover:bg-orange-500 undefined"
  >
    Add to cart
  </button>
</div>
`;

exports[`On render, check that Primary button list is rendered correctly 1`] = `
<div>
  <button
    class="text-sm rounded text-white p-3 cursor-pointer bg-hmred hover:bg-orange-500 undefined"
  >
    Add to cart
  </button>
</div>
`;
